src/main/java/nl/hsac/fitnesse/fixture/slim/FileFixture.java

```diff
@@ -49,6 +50,28 @@ public String textIn(String filename) throws IOException {
         return text;
     }
 
+    public String takeFirstLineFrom(String filename) throws IOException {
+        String result;
+        String fullName = getFullName(filename);
+        ensureParentExists(fullName);
+        File file = new File(fullName);
+        Scanner fileScanner = new Scanner(file);
+        if(fileScanner.hasNextLine()) {
+            result = fileScanner.nextLine();
+        } else{
+            throw new IOException(fullName + " is an empty file.");
+        }
+
+        //Overwrite file, minus the first line.
```

Comment:
Any chance you are willing to rewrite this to a (streaming) write to another file (e.g. .new) and at the end remove old file and rename new file to old name?
That way the entire file does not need to be loaded in memory and there is no chance of corrupting the existing file.

Maybe a bit overkill for the purpose at hand, but that just feels safer...